examples/nuclio/ingest/ingest_example.go

```diff
@@ -1,84 +1,186 @@
-package ingest
+package main
 
 import (
 	"encoding/json"
+	"os"
+	"sort"
+	"strings"
 	"sync"
 
 	"github.com/nuclio/nuclio-sdk-go"
-	"github.com/v3io/v3io-go-http"
+	"github.com/pkg/errors"
 	"github.com/v3io/v3io-tsdb/pkg/config"
 	"github.com/v3io/v3io-tsdb/pkg/tsdb"
-	"github.com/v3io/v3io-tsdb/pkg/tsdb/tsdbtest"
 	"github.com/v3io/v3io-tsdb/pkg/utils"
 )
 
-// Configuration
-// Note: the TSDB instance (`path`) must first be created using the CLI or API.
-// The user must also define the V3IO data binding in the Nuclio function - including path, username, and password - and name it "db0".
-var tsdbConfig = `
-path: "pmetric"
-`
-
-// Example event
-const pushEvent = `
-{
-  "Lset": { "__name__":"cpu", "os" : "win", "node" : "xyz123"},
-  "Time" : "now-5m",
-  "Value" : 3.7
+// Example event:
+//
+// {
+//		"metric": "cpu",
+//		"labels": {
+//			"dc": "7",
+//			"hostname": "mybesthost"
+//		},
+//		"samples": [
+//			{
+//				"t": "1532595945142",
+//				"v": {
+//					"N": 95.2
+//				}
+//			},
+//			{
+//				"t": "1532595948517",
+//				"v": {
+//					"n": 86.8
+//				}
+//			}
+//		]
+// }
+
+type value struct {
+	N float64 `json:"n,omitempty"`
```

Comment:
change to `interface{}` since we also support variant types.

Reply:
That's not how the json parser work. You can talk to Orit about changing the json format :)